romana/client/client.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
I think "client" is a very generic name, while this package here is about specific glue code for different environments. We used to call those 'environment adaptors'. I wonder if we could give this package here a more specific name?